### ISSUE TITLE
gridding loss

### ISSUE THREAD

**Issue by KLM1109** [hzxie/GRNet#54]
i'm a beginner in deeplearn, i can't find a place in the code to use gridding loss(extensions.gridding is exist, but it is not used ).can you tell me where to use it

**Comment by KLM1109** [comment]
sparse_loss = chamfer_dist(sparse_ptcloud, data['gtcloud'])
dense_loss = chamfer_dist(dense_ptcloud, data['gtcloud'])
grid_loss = gridding_loss(sparse_ptcloud, data['gtcloud'])
_loss = sparse_loss + dense_loss + grid_loss

is just modify here?thanks.

**Comment by hzxie** [comment]
Refer to #8.
